test/e2e/specs/onboarding/signup__with-theme-LOHP.ts

```diff
@@ -78,15 +78,16 @@ describe( 'Lifecyle: Logged Out Home Page, signup, onboard, launch and cancel su
 					themeButtonUrl.protocol = calypsoUrl.protocol;
 
 					await route.abort();
+					await page.waitForTimeout( 2000 );
```

Comment:
And this lets `abort()` finish, so no race condition with `page.goto()`... hell.